docs/root/source/data-models/block-model.rst

```diff
@@ -20,7 +20,7 @@ A block has the following structure:
 - ``id``: The hash of the serialized ``block`` (i.e. the ``timestamp``, ``transactions``, ``node_pubkey``, and ``voters``). This is also a database primary key; that's how we ensure that all blocks are unique.
 
 - ``block``:
-    - ``timestamp``: The Unix time when the block was created. It's provided by the node that created the block. See `the page about timestamps <https://docs.bigchaindb.com/en/latest/timestamps.html>`_.
+    - ``timestamp``: The Unix time when the block was created. It's provided by the node that created the block.
```

Comment:
I'd say the objective is to make clear that the node generated timestamps (and, really, any timestamps not verifiable) should not be trusted:

> The server-provided timestamps are still not 100% reliable or trustworthy. (A Byzantine node could put whatever timestamp it wants in its blocks or votes.)